internal/clientcache/internal/daemon/server.go

```diff
@@ -31,6 +31,16 @@ import (
 	"github.com/hashicorp/go-secure-stdlib/parseutil"
 )
 
+const (
+	// DefaultSearchStaleness is the default amount of time that can pass before
+	// a search request causes a blocking refresh to happen for the resources
+	// being searched for.
+	DefaultSearchStaleness = 60 * time.Second
+	// DefaultSearchRefreshTimeout is the default amount of time that a refresh
+	// that is blocking a search request can run before timing out.
+	DefaultSearchRefreshTimeout = 1 * time.Second
```

Comment:
Is this too low? Not sure if we've run benchmarks around how fast we can get a list from Boundary when refreshing but it would be a shame if Boundary is actually returning fresh results and we simply cut it off. Wonder if this should be somewhere between 1-2.